### PR TITLE
adds hammer and gavel

### PR DESCRIPTION
**Pull request by dannno**
it does what you think it does
no i didn't add it to the map

### CODE REVIEW

**Comment by dannno** [comment]
@Miauw62 if i got the spans wrong or whatever, outdated or some shit idj

File: code/game/objects/items/weapons/courtroom.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	icon = 'icons/obj/items.dmi'
+	icon_state = "gavelblock"
+	force = 2.0
+	throwforce = 2.0
+	w_class = 1.0
+
+/obj/item/weapon/gavelblock/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/weapon/gavelhammer))
+		playsound(loc, 'sound/items/gavel.ogg', 100, 1)
+		user.visible_message("<span class='notice'>[user] strikes \the [src] with the [I].</span>")
```
Comment (MrPerson) [inline code/game/objects/items/weapons/courtroom.dm]: \the [I] or else you'll get "Judge McGriffon strikes the gavel block with the the gavel."

**Comment by MrPerson** [comment]
Good idea, what kind of courtroom doesn't have a gavel?

File: code/game/objects/items/weapons/courtroom.dm (hunk 1 above)
Comment (dannno) [inline code/game/objects/items/weapons/courtroom.dm]: woopsy daisy

**Comment by pudl** [comment]
resprite the gavel block

**Comment by bgobandit** [comment]
does it do damage

**Comment by RemieRichards** [comment]
The block does 2 damage, the gavel does 5.

**Comment by Chocobro** [comment]
fan of the sprites :+1:

File: code/game/objects/items/weapons/courtroom.dm [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	icon = 'icons/obj/items.dmi'
+	icon_state = "gavelblock"
+	force = 2.0
+	throwforce = 2.0
+	w_class = 1.0
+
+/obj/item/weapon/gavelblock/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/weapon/gavelhammer))
+		playsound(loc, 'sound/items/gavel.ogg', 100, 1)
+		user.visible_message("<span class='notice'>[user] strikes \the [src] with \the [I].</span>")
```
Comment (Cheridan) [inline code/game/objects/items/weapons/courtroom.dm]: 'warning' is probably the more appropriate span here.

**Comment by Dorsisdwarf** [comment]
The gavel strike should be in big red text. it's for ORDAH IN THE COURT, is it not?

**Comment by pudl** [comment]
resprite the gavel block

**Comment by dannno** [comment]
what would i even change about it
it's a small circular piece of wood

File: html/changelogs/dannno-gavel.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+author: Dannno
+
+delete-after: True
+
+changes: 
+  - rscadd: "Added a hammer and gavel. Court is now in session."
```
Comment (RandomMarine) [inline html/changelogs/dannno-gavel.yml]: Should additions that players can't actually access have changelog entries?

**Comment by dannno** [comment]
block sprite changed in order to look different from an upcoming PR

File: html/changelogs/dannno-gavel.yml (hunk 3 above)
Comment (dannno) [inline html/changelogs/dannno-gavel.yml]: fuuuuuuck 'eeemmmmmmmm
in all seriousness hopefully a mapper will put them on the map

File: code/game/objects/items/weapons/courtroom.dm (hunk 2 above)
Comment (hornygranny) [inline code/game/objects/items/weapons/courtroom.dm]: ^

Comment (dannno) [inline code/game/objects/items/weapons/courtroom.dm]: woops didn't see this

Comment (dannno) [inline code/game/objects/items/weapons/courtroom.dm]: done

**Comment by Nienhaus** [comment]
@RemieRichards @Miauw62

**Comment by Miauw62** [comment]
Wow rude hg, nien asked for me or rr
Op 26-mrt.-2015 00:13 schreef "hornygranny" <notifications@github.com>:

**Comment by Nienhaus** [comment]
That was because you two were the first to come to mind.

**Pull request merged**